apps/src/code-studio/pd/professional_learning_landing/LandingPage.jsx

```diff
@@ -86,23 +85,24 @@ const getAvailableTabs = permissions => {
   return tabs;
 };
 
-const getEnrollSucessWorkshopName = () => {
+const getEnrollSucessWorkshopTitle = () => {
   // If sent here from successfully enrolling in a workshop, log WORKSHOP_ENROLLMENT_COMPLETED_EVENT.
-  const urlParams = queryParams();
-  if (urlParams && Object.keys(urlParams).includes('wsCourse')) {
-    const workshopCourseName = urlParams['wsCourse'];
+  const workshopCourse = sessionStorage.getItem('workshopCourse', null);
+  if (!workshopCourse) {
+    return '';
```

Comment:
Even with the name field, we're always expecting a `workshopCourse`?

Reply:
Yep! If a user has successfully enrolled, the one guaranteed field we will have is the course name. So if there is a course name then we know there's a new enrollment, and if not then we shouldn't show the enrollment success message.